GeoLib/AnalyticalGeometry.h

```diff
@@ -110,6 +125,16 @@ void computeRotationMatrixToXZ(MathLib::Vector3 const& plane_normal,
 /**
  * rotate points according to the rotation matrix
  * @param rot_mat 3x3 dimensional rotation matrix
+ * @param pnts_begin Iterator pointing to the initial element in a vector of points to be rotated
+ * @param pnts_end Iterator pointing to the element following the last element in a vector of points to be rotated
+ */
+void rotatePoints(
+        MathLib::DenseMatrix<double> const& rot_mat,
+        std::vector<GeoLib::Point*>::iterator pnts_begin, std::vector<GeoLib::Point*>::iterator pnts_end);
```

Comment:
Somewhat inconsistent interface compared to `getNewellPlane`: Here the iterator type is fixed, while in `getNewellPlane` it is a template type.